### ISSUE TITLE
uninitialized constant OpenStruct (NameError)

### ISSUE THREAD

**Issue by Hilaryous** [rails/jbuilder#282]
Getting this error from upgrading to 2.3.0

``` 
/home/ubuntu/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/jbuilder-2.3.0/lib/jbuilder.rb:26:in `<class:Jbuilder>': uninitialized constant OpenStruct (NameError)
        from /home/ubuntu/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/jbuilder-2.3.0/lib/jbuilder.rb:7:in `<top (required)>'
        from /home/ubuntu/.rbenv/versions/2.1.5/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:135:in `require'
        from /home/ubuntu/.rbenv/versions/2.1.5/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:135:in `rescue in require'
        from /home/ubuntu/.rbenv/versions/2.1.5/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:144:in `require'
        from lib/example.rb:1:in `<main>'
```

Did a small example to isolate the issue
Gemfile:
```ruby 
 source 'https://rubygems.org'
 gem 'jbuilder'
```

example.rb:
```ruby
 require 'jbuilder'
 
 puts "Hello World"
```
Then just ran ```ruby example.rb``` and got the aforementioned error.

**Comment by rwz** [comment]
Fixed in https://github.com/rails/jbuilder/pull/281

**Comment by rwz** [comment]
Released 2.3.1
